releases/client/2.3/settings

```diff
@@ -1,3 +1,3 @@
 FOREMAN_VERSION="2.3"
-FULLVERSION=$FOREMAN_VERSION
+FULLVERSION="2.3.0-RC1"
```

Comment:
Why? This was fine as it was IMHO. It's not actually used anyway and better than something which isn't true either.

Reply:
There is a step in the process where it expects the full version, not just the x.y, and it dies every time.

Reply:
Not for the client, no

Reply:
Then.... why does it fail and I have to make the edit. every. single. time.

Reply:
*what* does fail?

Reply:
Ah, release_pipeline script for client fails because ci-jobs expects the full version... Mhhh

Reply:
There ya go!  I was going through the rpm scripts but looks like you got to it.

Reply:
I still don't really see how that would fail. For the client pipeline a version with major.minor should suffice and we have that here.

Reply:
https://github.com/theforeman/theforeman-rel-eng/blob/622870650969aa3367e298d532db2f6230623b8b/ci-jobs#L130-L133

Reply:
Ah right. Instead of changing these settings files with confusing out of date info, I think we should instead change the parsing to handle major.minor.

Reply:
I prefer to merge https://github.com/theforeman/theforeman-rel-eng/pull/92 instead.